wedlocks/src/templates/welcome.js

```diff
@@ -0,0 +1,20 @@
+export default {
+  subject: () => 'Welcome to Unlock! Please, read this email carefuly',
+  text: params =>
+    `Welcome to Unlock! We're excited to have you with us!
+
+Unlock is designed to make sure we can never access your data: it's encrypted by your password. Because of that, we can't reset your password like other services that you're used to.
```

Comment:
```suggestion
Unlock is designed to make sure we can never access your data: it's encrypted with your password. Because of that, we can't reset your password like other services that you're used to.
```

This reads a bit more naturally to me